examples/purchaseTesterTypescript/app/components/CustomerInfoHeader.tsx

```diff
@@ -1,11 +1,8 @@
 import React, { useState } from 'react';
-import { Alert, Button, Modal, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { Button, Modal, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
 
-import { CustomerInfo, CustomerInfoUpdateListener } from 'react-native-purchases';
-
-import Purchases from 'react-native-purchases';
+import Purchases, { CustomerInfo } from 'react-native-purchases';
 import { presentPaywall } from 'react-native-purchases-ui';
-import { useRoute } from '@react-navigation/native';
```

Comment:
The changes in this screen are just indentation and import clean ups performed automatically by IntelliJ Idea, I thought it would be nice to commit them